### ISSUE TITLE
Using caffe in simple tasks.

### ISSUE THREAD

**Issue by inferrna** [BVLC/caffe#3233]
I compared caffe with neurolab in simple sine problem and neurolab looks better. Here my tries:

https://gist.github.com/inferrna/b3d7e162e6a6eb1ca974

nrsine.py - neurolab variant, gives mean error around 0.001 on test data.
trainsin_caffe.py - caffe variant (uses data prepared with convert2lmdb.py ), gives error around 0.02, on training stage it stagnates on 'loss = 0.000177079'
If I wrong somewhere, it would be great to add right example for simple cases like this (example for non-image data and non-integer label also would be helpful)
If I doing all right - caffe needs some fixes.

**Comment by longjon** [comment]
Closing as this looks like a request for help/general discussion (though I agree it would be nice to have some examples of this sort in Caffe).

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:
> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
